question/bank/exporttoxml/classes/exporttoxml_helper.php

```diff
@@ -0,0 +1,67 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Helper class for export to xml.
+ *
+ * @package    qbank_exporttoxml
+ * @copyright  2021 Catalyst IT Australia Pty Ltd
+ * @author     Safat Shahin <safatshahin@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace qbank_exporttoxml;
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * Class exporttoxml_helper
+ *
+ * @package    qbank_exporttoxml
+ * @copyright  2021 Catalyst IT Australia Pty Ltd
+ * @author     Safat Shahin <safatshahin@catalyst-au.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class exporttoxml_helper {
+
+    /**
+     * Get the URL to export a single question (exportone.php).
+     *
+     * @param \stdClass|\question_definition $question the question definition as obtained from
+     *      question_bank::load_question_data() or question_bank::make_question().
+     *      (Only ->id and ->contextid are used.)
+     * @return \moodle_url the requested URL.
+     */
+    public static function question_get_export_single_question_url($question) {
```

Comment:
can you please validate if is possible to add the return type here `: \moodle_url`